chain/messagepool/messagepool.go

```diff
@@ -355,31 +363,38 @@ func (mp *MessagePool) addLocal(m *types.SignedMessage, msgb []byte) error {
 	return nil
 }
 
-func (mp *MessagePool) verifyMsgBeforeAdd(m *types.SignedMessage, curTs *types.TipSet, local bool) error {
+func (mp *MessagePool) verifyMsgBeforeAdd(m *types.SignedMessage, curTs *types.TipSet, local bool) (bool, error) {
```

Comment:
I can get it from reading the function, but could I get a quick docstring here that explains briefly what the function does (mostly interested in what the return value means)

Reply:
sure, will do.

Reply:
done